CMakeLists.txt

```diff
@@ -93,3 +95,8 @@ TARGET_LINK_LIBRARIES(qlibc PRIVATE ${CMAKE_THREAD_LIBS_INIT})
 TARGET_LINK_LIBRARIES(qlibcext-static PRIVATE ${CMAKE_THREAD_LIBS_INIT})
 TARGET_LINK_LIBRARIES(qlibcext PUBLIC qlibc)
 
+INSTALL(DIRECTORY ${qlibc_SOURCE_DIR}/include/qlibc DESTINATION include)
+INSTALL(TARGETS   qlibc           LIBRARY           DESTINATION lib)
+INSTALL(TARGETS   qlibc-static    LIBRARY           DESTINATION lib)
+INSTALL(TARGETS   qlibcext        LIBRARY           DESTINATION lib)
+INSTALL(TARGETS   qlibcext-static LIBRARY           DESTINATION lib)
```

Comment:
Hi,  what do these lines do?

Reply:
Use configure  to compile
```shell
./configure --prefix=/tmp/libqlibc
make
make install # This step will install header and library files to /tmp/libqlibc

```

Use CMakeLists.txt to compile
```shell
mkdir build
cd build/
cmake -DCMAKE_INSTALL_PREFIX=/tmp/libqlibc ..
make
make install # This step will report an error
```

The function of these lines is that when using CMakeLists.txt to compile, "make install" can be executed normally.

Reply:
Thank you for the detailed explanations. Looks great!